config/daemonconfig.go

```diff
@@ -61,8 +61,12 @@ type DaemonConfig struct {
 }
 
 type MirrorConfig struct {
-	Host    string            `json:"host,omitempty"`
-	Headers map[string]string `json:"headers,omitempty"`
+	Host                string            `json:"host,omitempty"`
+	Headers             map[string]string `json:"headers,omitempty"`
+	AuthThrough         bool              `json:"auth_through,omitempty"`
+	HealthCheckInterval uint64            `json:"health_check_interval,omitempty"`
```

Comment:
Um...  uint64 is a considerable number. Is it necessary?

Reply:
However, it's not a blocker, let's merge it now

Reply:
No, It is not necessary, just to adapt the type in `image-service`.